src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -473,6 +472,30 @@ export default class AircraftModel {
             speed: 0
         };
 
+        /**
+         * @for AircraftModel
+         * @property model
+         * @type {AircraftTypeDefinitionModel}
+         * @private
+         */
+        this.model = new AircraftTypeDefinitionModel(options.model);
+
+        /**
+         * @for AircraftModel
+         * @property mcp
+         * @type {ModeController}
+         * @private
+         */
+        this.mcp = new ModeController();
+
+        /**
+         * @for AircraftModel
+         * @property fms
+         * @type {Fms}
+         * @private
+         */
+        this.fms = new Fms(options);
```

Comment:
These properties are not private

Reply:
They weren't before.

Reply:
Sorry, I mean to say that the docblocks show these as `@private`, and they shouldn't because they are public.

Reply:
done